packages/documentation/src/utils/index.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Siemens AG
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+export function capitalizeFirstLetter(name: string) {
+  if (name.length === 0) {
+    return name;
+  }
+  return name.charAt(0).toUpperCase() + name.slice(1);
```

Comment:
In order to take care to component that formatted according to kebab case
  return name
    .split('-')
    .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
    .join('');